tools/java/org/unicode/cldr/json/LdmlConvertRules.java

```diff
@@ -187,7 +213,15 @@
         "identity:language:type",
         "identity:script:type",
         "identity:territory:type",
-        "identity:variant:type");
+        "identity:variant:type"
+
+//        // TODO: Blocked by [CLDR-14272] - need more documentation
+//        "grammaticalFeatures:grammaticalGender:values",
```

Comment:
The grammaticalFeatures values are documented in https://unicode-org.github.io/cldr/ldml/tr35-general.html#Gender and beyond
The grammaticalDerivations values are documented in https://unicode-org.github.io/cldr/ldml/tr35-general.html#gender_compound_units and beyond.
If you think those need more information. please add that to https://unicode-org.atlassian.net/browse/CLDR-14272